src/components/Profile/OptionLinkWithIcon/OptionLinkWithIcon.styled.js

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+
+export const Section = styled.section`
+  margin-bottom: 20px;
+`;
+
+export const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 15px 0;
+
+  p {
```

Comment:
where are classes? why did you use tags?

Reply:
fix